features/step_definitions/authentication_steps.rb

```diff
@@ -93,10 +93,10 @@
 end
 
 Given /^I have a "([^\"]+)" cookie set to "([^\"]+)"$/ do |key, value|
-  headers = {}
-  Rack::Utils.set_cookie_header!(headers, key, value)
-  cookie_string = headers['Set-Cookie']
-  Capybara.current_session.driver.browser.set_cookie(cookie_string)
+    headers = {}
```

Comment:
looks like you introduced some extra spacing here - we're consistently on 2spaces, so would be great if we could stick to that :-)